### ISSUE TITLE
Navigation bar Inbox to another URL?

### ISSUE THREAD

**Issue by poehaldur** [woocommerce/woocommerce-admin#3966]
**Is your feature request related to a problem? Please describe.**
Really love the new WC navigation bar but I have no use for the inbox. 

**Describe the solution you'd like**
I have a “leads from forms” listings page, so I would like a way to change the inbox button URL to that page.

**Additional context**
![download (11)](https://user-images.githubusercontent.com/32192439/77315005-6edaee00-6d0f-11ea-9f54-034ecb3c8efc.png)

Long story short, I would like some help creating the inbox url filter to another URL.

I could not find the function in source code, is it js only?

**Comment by jameskoster** [comment]
We hope the Inbox will prove useful over time, so I wouldn't recommend replacing it.

Instead it sounds like it would be better for you to add a tab panel here. Hopefully an engineer will chime in with advice on whether that's possible and how to achieve it.

**Comment by poehaldur** [comment]
Thanks @jameskoster , I know it will be useful to beginners but I have clients who do not have access to settings as I take care of everything and to them inbox recommendations to connect with wordpress.com and etc. is just confusing. Even with new tab, I would still hide the inbox with css but you are right, that new tab would be a better solution than replacing inbox url.

**Comment by psealock** [comment]
>Instead it sounds like it would be better for you to add a tab panel here. Hopefully an engineer will chime in with advice on whether that's possible and how to achieve it.

I agree that would be useful. Unfortunately, we don't have filters in place to do that currently. I think it could add value and a discussion around what that would entail will be good.

**Comment by jameskoster** [comment]
Hey @poehaldur , without wanting to derail this too much, I just wanted to ask a couple of questions.

> I have clients who do not have access to settings as I take care of everything and to them inbox recommendations to connect with wordpress.com and etc. is just confusing.

How would you feel about the idea of allowing different users to toggle what category of notifications they receive, based on their role at the store? As a basic example, we could have notification categories like "Marketing", "Technology", "Education", etc. Merchants would likely be interested in educational material and ways to better market their stores. Developers are more likely to be interested in Woo tech. What do you think?
